backup-and-restore.adoc

```diff
@@ -0,0 +1,130 @@
+= Backup and restore Taiga
+:toc: left
+:toclevels: 1
+:numbered:
+:source-highlighter: pygments
+:pygments-style: friendly
+
+== Recommended backup process for Taiga6 docker
+
+This is the recommended backup process, as using Taiga6 with docker is the recommended way to run Taiga in production. This tutorial assumes that you have a Taiga6 docker instance running.
+
+This tutorial has been tested with link:https://github.com/kaleidos-ventures/taiga-docker[the official Taiga6 docker image].
+
+First, create a directory on the host where you want to store your backups in and switch into it. For this example we're going to use
+
+[source,bash]
+----
+:/# mkdir taiga-backup
+:/# cd taiga-backup
+----
+
+We have to find the container ID of the database container. To get a list of all running containers including their IDs, run
```

Comment:
using `taiga-docker` we don't need to find the container ID. As you use later (in the restore part), we can use the pre-defined name of the container (`taiga-docker_taiga-db_1`), so the user doesn't need to find the ID.

Reply:
Done ✅